infinispan-remote/cross-site-replication/minikube/setup.sh

```diff
@@ -0,0 +1,11 @@
+kubectl create -f identities_lon.yaml
+kubectl create -f identities_nyc.yaml
+
+kubectl apply -f LON.yaml
+kubectl apply -f NYC.yaml
+
+# TODO wait the cluster to form
+sleep 120
```

Comment:
You should be able to create the cache yamls at the same time as the other resources. The operator will ensure that the caches are only created once the cluster is available.

Similarly, you can wait for the cross-site view to be well formed with the following  commands:

```bash
kubectl -n <deployment-namespace> wait --for=condition=CrossSiteViewFormed infinispan example-cluster-lon
kubectl -n <deployment-namespace> wait --for=condition=CrossSiteViewFormed infinispan example-cluster-nyc
```